lib/services/presets.go

```diff
@@ -203,6 +204,7 @@ func defaultAllowRules() map[string][]types.Rule {
 			types.NewRule(types.KindSAMLIdPServiceProvider, RW()),
 			types.NewRule(types.KindOktaImportRule, RW()),
 			types.NewRule(types.KindOktaAssignment, RW()),
+			types.NewRule(types.KindCertAuthorityRevocationList, RW()),
```

Comment:
Can we also extend the buildIn RoleWindowsDesktop role permission: https://github.com/gravitational/teleport/blob/master/lib/authz/permissions.go#L776  ?